content/reference-rippled.md

```diff
@@ -1571,7 +1571,7 @@ The request includes the following parameters:
 | `Field`        | Type                                       | Description    |
 |:---------------|:-------------------------------------------|:---------------|
 | `account`      | String                                     | A unique identifier for the account, most commonly the account's address. |
-| `type`         | String                                     | _(Optional)_ If included, filter results to include only this type of ledger node. Valid types include `state` (trust lines), `offer` (offers), and `ticket` (part of the forthcoming signing process). |
+| `type`         | String                                     | _(Optional)_ If included, filter results to include only this type of ledger node. The valid types are: `offer`, `signer_list`, `state` (trust line), `escrow`, and `payment_channel`. <!-- Author's note: Omitted types from this list that can't be owned by an account: https://github.com/ripple/rippled/blob/1dbc5a57e6b0e90a9da0d6e56f2f5a99e6ac1d8c/src/ripple/rpc/impl/RPCHelpers.cpp#L676-L686 --> |
```

Comment:
Maybe this is already addressed in the other PR, but shouldn't "ledger node" be changed to "ledger object"?

Reply:
Yeah, the other PR isn't merged yet, but this would belong in that PR. (It's not there yet, though. So in that sense, this is a good catch in the wrong place.)